engli/baze.md

```diff
@@ -27,8 +27,7 @@ This document is a simple guide to the main features of Pandunia.
 | s | /s/ |
 | t | /tʰ/ |
 | u | /u/ |
-| v | /v/ |
-| w | /w~ʋ/ |
+| w | /w~v/ |
```

Comment:
Should be:
| v | /w~ʋ~v/ |

The labiodental approximant /ʋ/ is a likely realization of "v" in certain accents.

Reply:
I just fixed it.